docs/tutorial/part-three/index.md

```diff
@@ -4,54 +4,55 @@ typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Welcome to part three!
+ट्युटोरिअल पार्ट ३ मध्ये आपले स्वागत आहे.
 
-## What's in this tutorial?
+## आपण काय पाहणार आहोत ह्या ट्युटोरिअल मध्ये?
 
-In this part, you'll learn about Gatsby plugins and creating "layout" components.
+या भागात आपण गॅटस्बी प्लगइन्स आणि "लेआउट" कॉम्पोनन्ट तयार करण्याबद्दल जाणून घेऊयात.
 
-Gatsby plugins are JavaScript packages that help add functionality to a Gatsby site. Gatsby is designed to be extensible, which means plugins are able to extend and modify just about everything Gatsby does.
+गॅटस्बी प्लगइन्स Javascript पॅकेजेस आहेत जी गॅटस्बी साइटवर कार्यक्षमता जोडण्यास मदत करतात. गॅटस्बी एक्सटेंसिबल म्हणून डिझाइन केले गेले आहे, याचा अर्थ असा आहे की गॅट्सबी जे काही करतो त्याबद्दल प्लगइन केवळ विस्तारित आणि सुधारित करण्यास सक्षम आहेत.
```

Comment:
गॅट्सबी =>गॅटस्बी

Reply:
Please check. Now corrected.
Thanks for your review!